tests/integ/modin/window/test_rolling.py

```diff
@@ -142,6 +146,141 @@ def test_rolling_sem_ddof(ddof):
     )
 
 
+@sql_count_checker(query_count=1, join_count=1)
+def test_rolling_corr_simple():
+    native_df = native_pd.DataFrame({"col1": [1, 4, 3]})
+    other_native_df = native_pd.DataFrame({"col1": [1, 6, 3]})
+    snow_df = pd.DataFrame(native_df)
+    other_snow_df = pd.DataFrame(other_native_df)
+    snow_df = snow_df.rolling(window=3, min_periods=3).corr(
+        other=other_snow_df,
+        pairwise=None,
+        ddof=1,
+        numeric_only=True,
+    )
+    native_df = native_df.rolling(window=3, min_periods=3).corr(
+        other=other_native_df,
+        pairwise=None,
+        ddof=1,
+        numeric_only=True,
+    )
+    assert_snowpark_pandas_equals_to_pandas_without_dtypecheck(snow_df, native_df)
```

Comment:
Why not use `eval_snowpark_pandas_result` here and in other places?

Reply:
I was thinking it would be easier to directly compare the dfs which have as input 2 dataframes, similar to how it'd done in [test_join](https://github.com/snowflakedb/snowpark-python/blob/main/tests/integ/modin/frame/test_join.py#L43)